rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -75,15 +70,13 @@ def drawMol3D(m, view=None, confId=-1, drawAs=None, bgColor=None, size=None):
   if view is None:
     view = py3Dmol.view(width=size[0], height=size[1])
   view.removeAllModels()
+  
   try:
-    iter(m)
-  except TypeError:
-    addMolToView(m, view, confId, drawAs)
-  else:
-    ms = m
+    ms = iter(m)
     for m in ms:
       addMolToView(m, view, confId, drawAs)
-
+  except TypeError:
+    addMolToView(m, view, confId, drawAs)
```

Comment:
I wonder if this is a Python 2 left over. We should be able to write this now as:
```
from collections.abc import Iterable

if isinstance(m, Iterable):
  for mol in m:
    addMolToView(mol, view, confId, drawAs)
else:
  addMolToView(m, view, confId, drawAs)
```
or 
```
if not isinstance(m, Iterable):
  m = [m]
for mol in m:
  addMolToView(mol, view, confId, drawAs)
```
I checked that a Mol object is not an iterable.

Reply:
I don't agree with it. The main reason is that why we have to import some new modules to check whether it is iterable or not, or add to a list and push it to the function. I think that you can make another PR to fix it

Reply:
I am so sorry about that, but this would be unchanged. Thanks you for your review

Reply:
ok